### PR TITLE
Updated documentation with new sensor

### PR DESCRIPTION
**Pull request by jbarrancos**
**Description:**

Added rainsensor

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11023

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/rainbird.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: "Rain Bird"
+description: "Instructions on how to integrate your Rain Bird LNK WiFi Module within Home Assistant."
+date: 2017-08-25 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rainbird.png
+ha_category: Hub
+ha_release: 0.57
```
Comment (fabaff) [inline source/_components/rainbird.markdown]: Needs to updated after the merging in the main repo.

File: source/_components/rainbird.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+This `rainbird` component allows interacting with [LNK WiFi](http://www.rainbird.com/landscape/products/controllers/LNK-WiFi.htm) module of the Rain Bird Irrigation system in Home Assistant.
+
+To enable it, add the following to your `configuration.yaml` file:
+
+```yaml
+rainbird:
+    host: '1.1.1.1'
+    password: 'XXXXXXX'
```
Comment (fabaff) [inline source/_components/rainbird.markdown]: Please use two spaces for indent.

File: source/_components/sensor.rainbird.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+This `rainbird` sensor allows interacting with [LNK WiFi](http://www.rainbird.com/landscape/products/controllers/LNK-WiFi.htm) module of the Rain Bird Irrigation system in Home Assistant.
+
+Once you have enabled the [Rain Bird component](/components/rainburd), add the following to your `configuration.yaml` file to enable the rain sensor:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rainbird
+    monitored_conditions:
+    - rainsensor
```
Comment (fabaff) [inline source/_components/sensor.rainbird.markdown]: Wrong indent.

File: source/_components/rainbird.markdown [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+```yaml
+rainbird:
+  host: '1.1.1.1'
+  password: 'XXXXXXX'
+```
+
+Configuration variables:
+
+- **stickip** (*Required*): The IP address of your LNK WiFi Module.
```
Comment (armills) [inline source/_components/rainbird.markdown]: This should be `**host**`. Looks like an mistake from the old docs.

File: source/_components/sensor.rainbird.markdown (hunk 3 above)
Comment (armills) [reply source/_components/sensor.rainbird.markdown]: The line with `- rainsensor` needs two more spaces in front of it.

**Pull request merged**